package-lock.json

```diff
@@ -67,26 +67,6 @@
         "qs": "6.7.0",
         "raw-body": "2.4.0",
         "type-is": "~1.6.17"
-      },
```

Comment:
can you remove changes to package-lock.json? not 100% sure if they should be in the history.

Reply:
Yeah they should be, this is your version control and should always be up to date for if others are to clone the repo themselves